setup/helmRepository.yaml

```diff
@@ -1,3 +1,37 @@
+---
+apiVersion: v1
+kind: Pod
```

Comment:
Nit: Using a `Pod` directly will work with the given restartPolicy, but why not just use a Deployment?

Reply:
It's just an auto-generated yamls with the `kubectl run --restart Always --expose ...` command and I was a bit lazy to replace it with the Deployment 😄 Replaced with the Deployment